test/integration/testutils/utils.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2022 Google LLC
```

Comment:
```suggestion
// Copyright 2023 Google LLC
```

Reply:
Can we fix this on the PR https://github.com/GoogleCloudPlatform/terraform-google-enterprise-application/pull/61 ?
This build is green and ready to merge.

Reply:
Sure